ui/components/Sync/SuspendMessageModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { Dispatch, SetStateAction } from "react";
+import styled from "styled-components";
+import Button from "../Button";
+import Flex from "../Flex";
+import Modal from "../Modal";
+
+export type Props = {
+  onCloseModal: Dispatch<SetStateAction<boolean>>;
+  open: boolean;
+  setSuspendMessage: Dispatch<SetStateAction<string>>;
+  suspend: any;
+  suspendMessage: string;
+  className?: string;
+};
+
+const MessageModal = styled(Modal)`
+  textarea {
+    width: 100%;
+    box-sizing: border-box;
+    font-family: inherit;
+    font-size: 100%;
+    border-radius: ${(props) => props.theme.spacing.xxs};
+    resize: none;
+    margin-bottom: ${(props) => props.theme.spacing.base};
+  }
```

Comment:
Also we could add an `outline` with the standard light blue we've been using, rather than default-browser outline which looks a bit out of place

Reply:
nice idea